src/TodoList.js

```diff
@@ -3,23 +3,21 @@ import TodoListItem from './TodoListItem';
 
 
 //Add props as a parameter to the TodoList functional component
-function TodoList( {todoList} ) {
+/* Pass onRemoveTodo prop as a callback handler prop named onRemoveTodo to the TodoListItem component */
+function TodoList( {todoList, onRemoveTodo} ) {
 
 	//Change todoList to reference props instead of the hard-coded variable
 	//Deleted hard-coded todoList data
 
 	return (
 		<div>
 			<ul>
-
-			{ todoList.map((listItem) => {
-					return <TodoListItem key={listItem.id} item={listItem}/>;
-			})}
-			{/* you need the return inside map, because it is an anonymous function. if you don't have it, it implicitly return undefined. */}
-
+				{todoList.map((listItem) => {
+						return <TodoListItem key={listItem.id} item={listItem} onRemoveTodo={onRemoveTodo} />
+				})}
```

Comment:
I see your removed comment about the return inside the map.  You have to include the return if you include the `{}`.  Without them, there is an implied `return` statement.  So you could have this instead:

`todoList.map((listItem) => <TodoListItem key={listItem.id} item={listItem} onRemoveTodo={onRemoveTodo} />`